### DISCUSSION TITLE
Why is there no receive on-chain endpoint?

### DISCUSSION THREAD

**Discussion by secondl1ght** [ACINQ/phoenixd#38] [Q&A]
Just wondering why it is not possible to receive on-chain via the phoenixd API.

**Comment by xttlegendapi** [comment]
and why cant we choose a electrum server

**Reply by pm47** [reply]
Phoenixd uses mempool.space, not electrum. You can point it to your own mempool.space instance with `--mempool-space-url`

**Comment by pm47** [comment]
Duplicate of #24
